go-controller/pkg/ovn/egressip_test.go

```diff
@@ -1855,12 +1565,6 @@ var _ = ginkgo.Describe("OVN master EgressIP Operations", func() {
 				_, ipv4Sub, err := net.ParseCIDR(nodeIPv4)
 				_, ipv6Sub, err := net.ParseCIDR(nodeIPv6)
 
-				fakeOvn.fakeExec.AddFakeCmdsNoOutputNoError(
-					[]string{
-						fmt.Sprintf("ovn-nbctl --timeout=15 set logical_switch_port etor-GR_node1 options:nat-addresses=router"),
-					},
-				)
-
```

Comment:
Same as above. Please convert this check

Reply:
done